doc/contrib/index.rst

```diff
@@ -0,0 +1,101 @@
+.. _contribindex:
+
+Contributing to Arbor
+=====================
+
+First off, thank you for considering contributing to Arbor! It’s people
+like you that make Arbor such a great tool. Feel welcome and read the
+following sections in order to know how to ask questions and how to work
+on something.
+
+.. _contribindex-types:
+
+Types of contributions
+----------------------
+
+There are many ways to contribute: writing tutorials or blog posts,
+improving the documentation, helping other users, submitting bug reports
+and feature requests or writing code which can be incorporated into
+Arbor itself.
+
+If you come across a bug in Arbor, please `file an issue on the GitHub
+issue tracker <https://github.com/arbor-sim/arbor/issues/new>`__. You
+can attach files to issues. Data and bits of code that help to reproduce
```

Comment:
`Data and bits of code...` could change to: `Please include code samples and input data that help reproduce the issue; ...`